specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10292 @@
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}:listAuthKeys": {
+      "post": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Retrieves the authentication keys for an integration runtime.",
+        "operationId": "IntegrationRuntimes_ListAuthKeys",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/integrationRuntimeName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeAuthKeys"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_ListAuthKeys": {
+            "$ref": "./examples/IntegrationRuntimes_ListAuthKeys.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}:regenerateAuthKey": {
+      "post": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Regenerates the authentication key for an integration runtime.",
+        "operationId": "IntegrationRuntimes_RegenerateAuthKey",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/integrationRuntimeName",
+            "description": "The name of the integration runtime to retrieve."
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeRegenerateKeyParameters",
+              "description": "The request of regenerating auth key for integration runtime."
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeAuthKeys"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_RegenerateAuthKey": {
+            "$ref": "./examples/IntegrationRuntimes_RegenerateAuthKey.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}:disableInteractiveQuery": {
```

Comment:
Just checking: disableInteractiveQuery is not LRO, but enableInteractiveQuery is LRO?

Reply:
Thanks, I added long running for disable too

Reply:
Make sure you check the status code as well. If it is LRO, then status code won't be 200.